sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/hybridsearch/GlobalFullTextSearchQueryStatistics.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.query.hybridsearch;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.Document;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class GlobalFullTextSearchQueryStatistics {
+    @JsonProperty(Constants.Properties.DOCUMENT_COUNT)
+    private Long documentCount;
+    @JsonProperty(Constants.Properties.FULL_TEXT_QUERY_STATISTICS)
+    private List<FullTextQueryStatistics> fullTextQueryStatistics;
+    private JsonSerializable jsonSerializable;
```

Comment:
Why do you need tehJsonSerialiyable here?

Reply:
Removed it.